### PR TITLE
Fix for gorod.mos.ru

### PR DESCRIPTION
**Pull request by rakleed**
- Invert logo.
- Invert map.
- Invert footer.

Example of site pages:
- https://gorod.mos.ru/index.php
- https://gorod.mos.ru/?show=problem&zone=1
- https://gorod.mos.ru/index.php?show=problem
- https://gorod.mos.ru/?show=objects&action=show&id=356648&addr=#comments

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -6129,6 +6129,20 @@ INVERT
...
+gorod.mos.ru
+
+INVERT
+.scene__logo
+div[id^="map"] > div > div > div
+div[style^="text-align: center; margin"] > div > div:first-of-type
+
+CSS
+#footer {
+    background: #162F29 !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:6141]: Could this be in the style of `${}`?

Comment (rakleed) [reply src/config/dynamic-theme-fixes.config:6141]: And what is it for? And like `${#CCECE8}` is fine.

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:6141]: > And what is it for?

To ensure that different user settings will get the correct dynamic color.


> And like `${#CCECE8}` is fine.

Yap that's fine.

**Review by Gusted — APPROVED** [review]

**Comment by Gusted** [comment]
Their is a conflict going on.

**Comment by rakleed** [comment]
> Their is a conflict going on.

What do you have in mind?

**Comment by Gusted** [comment]
> What do you have in mind?

The easiest path is to just copy this fix and open up a new PR for it, as going around and resolve conflicts can be a pain.

**Comment by rakleed** [comment]
Done: https://github.com/darkreader/darkreader/pull/6979

**Pull request closed without merging**